test/erc677/ERC677.test.ts

```diff
@@ -0,0 +1,110 @@
+import {deployments, ethers, getNamedAccounts} from 'hardhat';
+import {BigNumber} from '@ethersproject/bignumber';
+import {expect} from '../chai-setup';
+import {Contract} from 'ethers';
+
+describe('ERC677Token', function () {
+  let tokenReceiver: Contract;
+  let gemToken: Contract;
+  let emptyContract: Contract;
+  let fallbackContract: Contract;
+
+  async function createFixture() {
+    await deployments.fixture();
```

Comment:
we should use tag in test tospeed them up when run individually
so here : `await deployments.fixture('Gems');